nf_core/pipeline-template/{{cookiecutter.pipeline_slug}}/docs/configuration/adding_your_own.md

```diff
@@ -43,10 +43,10 @@ First, install docker on your system: [Docker Installation Instructions](https:/
 
 Then, simply run the analysis pipeline:
 ```bash
-nextflow run nf-core/{{ cookiecutter.pipeline_name }} -profile docker --reads '<path to your reads>'
+nextflow run {{ cookiecutter.github_repo }} -profile docker --reads '<path to your reads>'
```

Comment:
All of the changes in this file are from a previous version of tools, so I guess it was a merge conflict gone wrong or something. But yeah, the main `nf-core/tools` code is correct, so please update your fork so that these changes disappear :)